pkgs/by-name/bi/biber-for-tectonic/package.nix

```diff
@@ -0,0 +1,4 @@
+/**
+  refer to `pkgs.tectonic` for the definition
+*/
+{ tectonic }: tectonic.biber
```

Comment:
By `pkgs/by-name` stubs I mean e.g. this

Reply:
> By `pkgs/by-name` stubs I mean e.g. this

Yea I understand :) :+1: .